pages/Category.vue

```diff
@@ -530,6 +530,7 @@ export default {
       }
     },
     changeFilter (filter) {
+      delete this.$route.query.page
```

Comment:
I think it is not a proper approach. You shouldn't modify this.$route.query object this way. Only via `push` or `replace` method. Please think about a different approach

Reply:
@Fifciu Solved the issue by updating the filterHelpers.ts file in catalog-next module.
PR: https://github.com/vuestorefront/vue-storefront/pull/5396